python/py-skrf/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        scikit-rf scikit-rf 0.15.3 v
+
+checksums           rmd160  33172e881c9b13d464a3e4bf0a5329c60ac1b188 \
+                    sha256  c0f84010eec1ac07755c9fb8e51295a0116037f2de0a2117f507f7f1848106da \
+                    size    17777889
+
+name                py-skrf
```

Comment:
please name the port `py-scikit-rf` to follow PyPI and be consistent with other `py-scikit` ports (this means also renaming the directory where the Portfile is in)